lib/src/value/calculation.dart

```diff
@@ -120,6 +122,228 @@ class SassCalculation extends Value {
+  /// [SassNumber] rather than a [SassCalculation]. It throws an exception if it
+  /// can determine that the calculation will definitely produce invalid CSS.
+  static Value asin(Object argument) {
+    argument = _simplify(argument);
+    if (argument is! SassNumber || argument.hasUnit('%')) {
+      return SassCalculation._("asin", [argument]);
+    }
+    return number_lib.asin(argument);
+  }
+
+  /// Creates a `acos()` calculation with the given [argument].
+  ///
+  /// The [argument] must be either a [SassNumber], a [SassCalculation], an
+  /// unquoted [SassString], a [CalculationOperation], or a
+  /// [CalculationInterpolation].
+  ///
+  /// This automatically simplifies the calculation, so it may return a
+  /// [SassNumber] rather than a [SassCalculation]. It throws an exception if it
+  /// can determine that the calculation will definitely produce invalid CSS.
+  static Value acos(Object argument) {
+    argument = _simplify(argument);
+    if (argument is! SassNumber || argument.hasUnit('%')) {
+      return SassCalculation._("acos", [argument]);
+    }
+    return number_lib.acos(argument);
+  }
+
+  /// Creates a `abs()` calculation with the given [argument].
+  ///
+  /// The [argument] must be either a [SassNumber], a [SassCalculation], an
+  /// unquoted [SassString], a [CalculationOperation], or a
+  /// [CalculationInterpolation].
+  ///
+  /// This automatically simplifies the calculation, so it may return a
+  /// [SassNumber] rather than a [SassCalculation]. It throws an exception if it
+  /// can determine that the calculation will definitely produce invalid CSS.
+  static Value abs(Object argument) {
+    argument = _simplify(argument);
+    if (argument is! SassNumber) {
+      return SassCalculation._("abs", [argument]);
+    }
+    if (argument.hasUnit("%")) {
+      warnForDeprecation(
+          "abs() will not longer support percentage units."
+          "future release.\n"
+          "\n"
+          "Recommendation: "
+          "abs(#{$argument})\n"
+          "\n"
+          "To preserve current behavior: "
+          "math.abs($argument)\n"
+          "More info: https://sass-lang.com/documentation/values/numbers",
+          Deprecation.absPercent);
+    }
+    return number_lib.abs(argument);
+  }
+
+  /// Creates a `exp()` calculation with the given [argument].
+  ///
+  /// The [argument] must be either a [SassNumber], a [SassCalculation], an
+  /// unquoted [SassString], a [CalculationOperation], or a
+  /// [CalculationInterpolation].
+  ///
+  /// This automatically simplifies the calculation, so it may return a
+  /// [SassNumber] rather than a [SassCalculation]. It throws an exception if it
+  /// can determine that the calculation will definitely produce invalid CSS.
+  static Value exp(Object argument) {
+    argument = _simplify(argument);
+    if (argument is! SassNumber || argument.hasUnit("%")) {
+      return SassCalculation._("pow", [number_lib.e(), argument]);
```

Comment:
This should emit an `exp()` CSS function, not a `pow`, because this would then compile to to `pow(2.71828182845, $argument)` rather than `exp($argument)`.

Reply:
In [the spec](https://github.com/sass/sass/blob/main/accepted/calc-functions.md#simplifying-a-calculation) it mentions `If calc's name is "exp" and arguments contains exactly a single number number with known units, return the result of calling math.pow(math.$e, number).` That is what we are doing here right? 🤔

Reply:
Oh I see, thanks :)